config/peerpods/podvm/README.md

```diff
@@ -34,3 +34,33 @@ Now when you create a KataConfig with `enablePeerPods: true` with empty
 `AZURE_IMAGE_ID` or `AWS_AMI_ID` in `peer-pods-cm`, then depending on the cloud
 provider configured, the operator will create the pod VM image based on the
 provided config.
+
+## PodVM Image Upload Configuration
+
+The PodVM image can be embedded into a container image. This container image can then be unwrapped and uploaded to the libvirt volume specified in the `peer-pods-secret`. Please note that this feature is currently supported only for the libvirt provider.
+
+To create an OCI image with the PodVM image, you can use the `Dockerfile.podvm-oci` as follows:
+
+```bash
+docker build -t podvm-libvirt \
+    --build-arg PODVM_IMAGE_SRC=<podvm_image_source> \
+    -f Dockerfile.podvm-oci .
+```
+
+In this context, `PODVM_IMAGE_SRC` refers to the location of the `qcow2` image on the host. Optionally, you can also set `PODVM_IMAGE_PATH`, which is the path of the qcow2 image inside the container. This path will be used as `<image_path>` in the `PODVM_IMAGE_URI` as described below.
+
+Ensure that `PODVM_IMAGE_URI` is configured in the `libvirt-podvm-image-cm` in the following format:
+
+```bash
+PODVM_IMAGE_URI: "<image_type>::<image_url>:<image_tag>::<image_path>"
```

Comment:
nit: image_type  can be changed to image_repo_type (to distinguish between operator-build and pre-built)
likewise image_url -> image_repo_url

Reply:
incorporated..